### DISCUSSION TITLE
is there any auto-completion?

### DISCUSSION THREAD

**Discussion by rudde0** [ShaneBeee/SkBee#712] [Q&A]
Before MC 1.20.5, I was returning my items with `string tag "OzelEsya" of nbt of player's held item` expression.

My question is, am I have to do add `minecraft:custom_data;` to beginning of the string every single time in newer MC versions?

I can get OzelEsya data still using `string tag "minecraft:custom_data;OzelEsya" of nbt of player's held item` but string is _unnecessarily(?)_ longer than usual.

check tag:
![image](https://github.com/user-attachments/assets/b299aba2-a432-4473-8274-f6ff62949f0a)

full nbt (not complete):
![image](https://github.com/user-attachments/assets/f07fae76-63f8-42fc-84aa-47134d1c0d96)

nbt (not complete):
![image](https://github.com/user-attachments/assets/afb8d284-7e38-4f60-92ab-6c4c7da0df94)

----

When I use `#set string tag "OzelEsya" of nbt of held item to "Oyun_Kiran"` nothing happened on the item, so I added this string to custom data with `#set string tag "minecraft:custom_data;OzelEsya" of nbt of held item to "Oyun_Kiran"`.

**Comment by ShaneBeee** [comment]
every single SkBee update since 1.20.5 has been released, has included details of the changes to how SkBee handles NBT.
There is also a link to a wiki in those posts that further explains in greater detail all the changes and what you will need to do on your end.
I recommend reading those changelogs and wikis.
